include/build_config/SDL_build_config.h.cmake

```diff
@@ -144,6 +129,7 @@
 #cmakedefine HAVE_CEILF 1
 #cmakedefine HAVE_COPYSIGN 1
 #cmakedefine HAVE_COPYSIGNF 1
+#cmakedefine HAVE__COPYSIGN 1
```

Comment:
Should `_copysign` be tested in `CMakeLists.txt`?

Reply:
Yes, it's available in Windows builds.